lib/plugins/leetcode.js

```diff
@@ -137,6 +137,7 @@ plugin.getProblem = function(problem, cb) {
       '    sampleTestCase',
       '    enableRunCode',
       '    metaData',
+      '    translatedContent',
```

Comment:
Thanks @EINDEX!

I assume that this new field would work for both leetcode.com and leetcode-cn.com?

Reply:
if the problems doesn't have chinese in leetcode-cn.com like [this sum-of-distances-in-tree](https://leetcode-cn.com/problems/sum-of-distances-in-tree/description/). The tranlatedContent filed is null.